tests/migration_test.go

```diff
@@ -1332,7 +1330,7 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse(fedoraVMSize)
 				vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
 
-				tests.AddUserData(vmi, "cloud-init", tests.GetFedoraToolsGuestAgentUserData())
+				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\n echo hello\n")
```

Comment:
do we still need echo hello ?
i remember it was adding some delay for ping on some tests, not sure if it still needed?

Reply:
I don't think so; iirc, the reason was that you *had* to provision **something** in the user data for cloud-init to do its thing. I don't remember the whole story though...

Reply:
Yes, it is not needed. I just kept it to stop refactoring at some point.

Reply:
In this case it is needed because we want to test migrations with a user-data disk.

Reply:
Can you please change the places where its needed to have it, to something like
`echo migration with user-data` or so, so if we would do in the future some updates, it will be clear why it was left ?